### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/c39321292f8b8568573876e71632a990da2f856a

### CODE REVIEW

**Review by peiying2 — COMMENTED** [review]
Only a minor comment on the punctuation with `in addition to` and where it's used.

**Comment by flodolo** [comment]
> @lucasferrazlima Only a minor comment on the punctuation with `in addition to` and where it's used.

There is no comment?

File: en/firefox/developer.ftl [hunk 1]
```diff
@@ -80,7 +77,11 @@ firefox-developer-congrats-you-now-have-firefox = Congrats. You now have { -bran
 firefox-developer-this-isnt-just-an-update = This isn’t just an update. This is { -brand-name-firefox-quantum }: A brand new { -brand-name-firefox } that has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
 firefox-developer-welcome-to-firefox-browser = Welcome to { -brand-name-firefox-browser } { -brand-name-developer-edition }
 firefox-developer-made-for-developers = The browser made for developers
+firefox-developer-all-the-latest-v2 = All the latest developer tools in beta, in addition to features like the Multi-line Console Editor and WebSocket Inspector.
```
Comment (peiying2) [inline en/firefox/developer.ftl:80]: Remove the comma before `in addition to features ...`

Comment (lucasferrazlima) [reply en/firefox/developer.ftl:80]: Thanks for the correction!
Can I simply open a new PR in the bedrock repository to meet your demand? Or is there some other way in which this fixing process is done?

Comment (peiying2) [reply en/firefox/developer.ftl:80]: If you fix it on the Bedrock side, that should trigger changes here, generating another commit to the same PR.

**Comment by flodolo** [comment]
(removed the branch by mistake)

**Review by peiying2 — APPROVED** [review]

**Pull request merged**